<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>535</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>640</width>
      <height>460</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listSortingAlgorithms">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>165</width>
      <height>240</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Bubble sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insertion sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Selection sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Merge sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quick sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heap sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shell sort</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pbStartStop">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>425</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbReset">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>425</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelArrayAccesses">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Array accesses:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelArrayAccessesVar">
    <property name="geometry">
     <rect>
      <x>765</x>
      <y>0</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderDelay">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>165</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLoadArrayFromFile">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>165</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load array from file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEnterArrayElement">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter array element:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEnterArrayElement">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelDelay">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delay:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddElements">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>165</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add elements</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
